app/models/project.rb

```diff
@@ -374,4 +377,8 @@ def has_minimum_goal_for_video?
   def reject_project_image(attributes)
     new_record? && attributes[:original_image_url].nil?
   end
+
+  def can_has_associated_plans?
```

Comment:
I just asked you for a name suggestion, u know? 😄 
what u thought about `permit_association_with_plans?`  ?